### PR TITLE
Turn Sentry Off by using SENTRY_ENABLE FLAG instead of SENTRY_DSN#12422

### PR DESCRIPTION
**Pull request by stevenc987**
Signed-off-by: Steven Chen <steven.chen@gov.bc.ca>

*Issue #:*
https://github.com/bcgov/entity/issues/12422

*Description of changes:*
Turn Sentry Off by using SENTRY_ENABLE FLAG instead of SENTRY_DSN#12422

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by seeker25** [comment]
Is this a hotfix? or can this be deployed next week?

File: auth-web/public/config/configuration.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "AFFIDAVIT_FILE_SIZE": "73KB",
   "PAYMENT_PAYEE_NAME": "BC Registries and Online Services",
   "SENTRY_DSN": "",
-  "SENTRY_ENABLE": "false",
+  "SENTRY_ENABLE": "False",
```
Comment (seeker25) [inline auth-web/public/config/configuration.json:16]: I'd recommend converting to lowercase and then comparing.

File: auth-api/src/auth_api/__init__.py [hunk 2]
```diff
@@ -44,11 +44,12 @@ def create_app(run_mode=os.getenv('FLASK_ENV', 'production')):
     app.config.from_object(config.CONFIGURATION[run_mode])
 
     # Configure Sentry
-    if app.config.get('SENTRY_DSN', None):
-        sentry_sdk.init(  # pylint: disable=abstract-class-instantiated
-            dsn=app.config.get('SENTRY_DSN'),
-            integrations=[FlaskIntegration()]
-        )
+    if app.config.get('SENTRY_ENABLE') == 'True':
```
Comment (seeker25) [inline auth-api/src/auth_api/__init__.py:47]: Convert to lowercase, then compare.

File: status-api/src/status_api/__init__.py [hunk 3]
```diff
@@ -38,11 +38,12 @@ def create_app(run_mode=os.getenv('FLASK_ENV', 'production')):
     app.config.from_object(CONFIGURATION[run_mode])
 
     # Configure Sentry
-    if app.config.get('SENTRY_DSN', None):
-        sentry_sdk.init(
-            dsn=app.config.get('SENTRY_DSN'),
-            integrations=[FlaskIntegration()]
-        )
+    if app.config.get('SENTRY_ENABLE') == 'True':
```
Comment (seeker25) [inline status-api/src/status_api/__init__.py:41]: Convert to lower case, then compare.

File: auth-web/public/config/configuration.json (hunk 1 above)
Comment (seeker25) [reply auth-web/public/config/configuration.json:16]: Otherwise if someone switches from False -> true, it might not work the way they'd expect.

**Comment by stevenc987** [comment]
@seeker25 
This can be deployed next week. I have updated codes as your suggestion. Could you please review them? Thanks

**Comment by seeker25** [comment]
Yup, just give me a shout when you need it merged.

**Pull request closed without merging**